phpstan/phpstan/php-82.neon

```diff
@@ -0,0 +1,13 @@
+parameters:
+    ignoreErrors:
+        # In PHP 8.2, PHPStan no longer believes @property works without magic methods,
+        # but all implementors of the interfaces actually have those properties.
+        - "~Access to an undefined property .*GraphQL\\\\Type\\\\Definition\\\\NamedType.*::\\$name~"
+        - "~Access to an undefined property .*GraphQL\\\\Type\\\\Definition\\\\NamedType.*::\\$description~"
+        - "~Access to an undefined property .*GraphQL\\\\Type\\\\Definition\\\\NamedType.*::\\$astNode~"
+        - "~Access to an undefined property .*GraphQL\\\\Type\\\\Definition\\\\NamedType.*::\\$extensionASTNodes~"
+        - "~Access to an undefined property .*GraphQL\\\\Language\\\\AST\\\\TypeDefinitionNode.*::\\$name~"
+        - "~Access to an undefined property .*GraphQL\\\\Language\\\\AST\\\\TypeExtensionNode.*::\\$name~"
+
+        # In PHP 8.2, PHPStan seems to not allow dynamic checks for properties on classes, even when they are not final.
+        - "~Call to function property_exists\\(\\) with .* will always evaluate to false\\.~"
```

Comment:
I think this is or will be fixed: https://github.com/phpstan/phpstan/issues/8474

Reply:
We can then remove the ignores but I'd keep this _version-based include framework_ so we can handle similar issues more easily in the future without having `reportUnmatched: false`.

Reply:
In order to avoid this

<img width="926" alt="image" src="https://user-images.githubusercontent.com/327717/215492301-1cffcef2-3d4c-4a0c-abc1-efe2024741aa.png">

Reply:
sounds good